backend/apps/volontulo/serializers.py

```diff
@@ -206,6 +211,10 @@ def get_organizations(self, obj):
         qs = obj.userprofile.organizations.all()
         return OrganizationSerializer(qs, many=True, context=self.context).data
 
+    def get_phone_no(obj):
```

Comment:
there's still `staticmethod` decorator missing

Reply:
+ `The docstring is a phrase ending in a period.` -> https://www.python.org/dev/peps/pep-0257/